data/pigui/modules/new-game-window/summary.lua

```diff
@@ -0,0 +1,105 @@
+-- Copyright © 2008-2023 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local ui = require 'pigui'
+local Crew = require 'pigui.modules.new-game-window.crew'
+local Ship = require 'pigui.modules.new-game-window.ship'
+local Defs = require 'pigui.modules.new-game-window.defs'
+local Format = require 'Format'
+local Widgets = require 'pigui.modules.new-game-window.widgets'
+local ShipDef = require 'ShipDef'
+local Lang = require 'Lang'
+local lui = Lang.GetResource("ui-core")
+local leq = Lang.GetResource("equipment-core")
+
+local Summary = {}
+
+local layout = {}
+
+local startDesc = ""
+
+function Summary:updateLayout()
+	layout.picsize = Defs.mainFont.size * 12
+	layout.leftWidth = math.round(0.6 * Defs.contentRegion.x)
+	layout.rightWidth = Defs.contentRegion.x - layout.leftWidth
+	layout.playerParam = {}
+	layout.shipParam = {}
+	layout.paramsGroup = {
+		child_id = "params",
+		width = layout.leftWidth - layout.picsize - Defs.gap.x,
+		height = layout.picsize
+	}
+end
+
+function Summary:draw()
+	local player = Crew.Player.Char.value
+
+	ui.child("leftside", Vector2(layout.leftWidth, Defs.contentRegion.y), function()
+		local player_member = Crew.Player.Char.crewEntry
+		Crew:initMemberFace(player_member)
+		ui.child("face", Vector2(layout.picsize, layout.picsize), function()
+			player_member.face:renderFaceDisplay()
+		end)
+		ui.sameLine()
+		Widgets.centeredIn(layout.paramsGroup, function()
+			ui.beginGroup()
+			Widgets.alignLabel(lui.NAME_PERSON, layout.playerParam, function()
+				ui.text(player.name)
+			end)
+			Widgets.alignLabel(lui.CASH, layout.playerParam, function()
+				ui.text(Format.Money(Crew.Player.Money.value, false))
+			end)
+			Widgets.alignLabel(lui.SHIP_TYPE, layout.playerParam, function()
+				ui.text(ShipDef[Ship.Type.value].name)
+			end)
+			ui.endGroup()
```

Comment:
The summary page should probably display the ship registration number or name, as well as the ship type IMO.

Reply:
Ship name is now shown.